applications/crossbar/src/modules_v2/cb_phone_numbers_v2.erl

```diff
@@ -29,6 +29,7 @@
 
 -define(CB_LIST, <<"phone_numbers/crossbar_listing">>).
 -define(PORT_NUM_LISTING, <<"port_requests/phone_numbers_listing">>).
+-define(KEY_PORT_NUMBER, 2).
```

Comment:
@icehess sorry, but can we rename it to `KEY_PORT_NUMBER_IDX_KEY` ? i think its more meaningful.